src/action.ts

```diff
@@ -41,22 +41,30 @@ export async function performAction(
 	tracker: string
 ): Promise<ActionResult> {
 	const { action, linkDir } = getRuntimeConfig();
-
+	let trackerLinkDir: string;
+	if (linkDir) {
+		trackerLinkDir = join(linkDir, tracker);
+	}
```

Comment:
small nit, using a ternary here allows this to be marked const:

```suggestion
	const trackerLinkDir = linkDir ? join(linkDir, tracker) : undefined;
	}
```